src/sidebar/util/test/sentry-test.js

```diff
@@ -133,7 +133,7 @@ describe('sidebar/util/sentry', () => {
         .getCalls()
         .find(call => call.args[0] === 'loaded_scripts').args[1];
       assert.isTrue(urls.length > 0);
-      urls.forEach(url => assert.match(url, /<inline>|http:.*\.js/));
+      urls.forEach(url => assert.match(url, /<inline>|http:.*/));
```

Comment:
Vitest injects one extra script that does not have the `.js` extension. I reckon this test is reliable enough like this, and then it passes with both karma and vitest, but we could also address the fixing directly in the vitest migration PR.